WalletWasabi.Tests/UnitTests/Tor/TorBinaryHashesTests.cs

```diff
@@ -18,9 +18,9 @@ public void VerifyTorBinaryChecksumHashes()
 
 			Dictionary<OSPlatform, string> expectedHashes = new()
 			{
-				{ OSPlatform.Windows, "d244a89f7ca9da2259925affa0bd008d3cdea1d9dfd24cf53efffb7c1eadc169" },
-				{ OSPlatform.Linux, "af7302d62fc1e47f79af8860541365f77547233404302a1e601e1f367e6e2888" },
-				{ OSPlatform.OSX, "fe6d719e18bf3a963f0274de259b7e029f40e4fe778f4d170bba343eb491af00" },
+				{ OSPlatform.Windows, "886960e5698cd275f81e2ad3b9654275172a30f6e9f16ddf09b6b96ad4b50403" },
```

Comment:
I would add a comment for this test saying which version of Tor we use. Something along the lines:

```csharp
/// <summary>
/// Verifies Tor executable file hashes (i.e. <c>tor.exe</c>, <c>tor.real</c> and <c>tor</c>) from Tor Browser version 10.0.10.
/// </summary>
```

Reply:
I am afraid it will get removed as a maintenance burden. https://github.com/zkSNACKs/WalletWasabi/pull/5074/commits/e2f1de00b1796c0d7dd145203d5c338f96862781

Reply:
I find it useful but ok.